spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/ConfigCheck.scala

```diff
@@ -0,0 +1,90 @@
+package com.datastax.spark.connector.util
+
+import com.datastax.spark.connector.cql.CassandraConnectorConf
+import com.datastax.spark.connector.rdd.ReadConf
+import com.datastax.spark.connector.writer.WriteConf
+import org.apache.commons.configuration.ConfigurationException
+import org.apache.commons.lang3.StringUtils
+import org.apache.spark.SparkConf
+
+/**
+ * Helper class to throw exceptions if there are environment variables in the spark.cassandra
+ * namespace which don't map to Spark Cassandra Connector known properties.
+ */
+object ConfigCheck {
+
+  val MatchThreshold = 0.85
+  val Prefix = "spark.cassandra."
+
+  val validProps =
+    WriteConf.Properties ++
+    ReadConf.Properties ++
+    CassandraConnectorConf.Properties
+
+
+  /**
+   * Checks the SparkConf Object for any unknown spark.cassandra.* properties and throws an exception
+   * with suggestions if an unknown property is found.
+   * @param conf SparkConf object to check
+   */
+  def checkConfig(conf: SparkConf): Unit = {
+    val unknownProps = getUnknownProperties(conf)
+    if (unknownProps.nonEmpty) {
+      val suggestions = for (unknownVar <- unknownProps; s <- getSuggestedVars(unknownVar)) yield (unknownVar, s)
```

Comment:
Please try to keep lines short. We have no formal style guide yet, but we typically try to stay within 80-100 chars with a hard limit to 120 chars. This improves readability and makes it easier to view code on GitHub.